Trend Micro/deep-security/ingest/parser.yml

```
@@ -42,10 +42,12 @@ stages:
           trendmicro.FragmentationBits: '{{parsed_event.message.get("Fragmentation Bits")}}'
           trendmicro.TCPFlags: '{{parsed_event.message.get("TCP Flags").split(" ")}}'
           trendmicro.TrendMicroDsRelevantDetectionNames: '{{parsed_event.message.TrendMicroDsRelevantDetectionNames.split(";")}}'
+          trendmicro.ResourceTypeId: '{{parsed_event.message.get("Resource Type")}}'
           container.name:  '{{parsed_event.message.get("Container")}}'
           file.name: '{{parsed_event.message.get("Infected Resource")}}'
           host.id: '{{parsed_event.message.get("Host ID")}}'
           event.reason: '{{parsed_event.message.get("actionReason")}}'
+          event.severity: 'parsed_event.message.get("Risk Level")'
```

The jinja brace are missing.
To respect the ECS specification, [`event.severity`](https://www.elastic.co/guide/en/ecs/8.4/ecs-event.html#field-event-severity) should be a long
```suggestion
          event.severity: '{{parsed_event.message.get("Risk Level") | int}}'
```

---

Good catch! thank you